pandas/core/groupby/groupby.py

```diff
@@ -328,7 +328,7 @@ class providing the base-class of operations.
 engine : str, default None
     * ``'cython'`` : Runs the function through C-extensions from cython.
     * ``'numba'`` : Runs the function through JIT compiled code from numba.
-    * ``None`` : Defaults to ``'cython'`` or globally setting ``compute.use_numba``
+    * ``None`` : Defaults to ``'cython'`` or the globally setting ``compute.use_numba``
```

Comment:
```suggestion
    * ``None`` : Defaults to ``'cython'`` or the global setting ``compute.use_numba``
```

Reply:
I've made the two commits with grammar fixes.